distribution/docs/src/main/resources/content/_configuring/map-layers-intrigue.adoc

```diff
@@ -8,6 +8,8 @@
 == {title}
 
 Customize the look of the map displayed to users in ${catalog-ui} by adding or removing map layers through the *Map Layers UI*.
+Equivalent addition, editing, and deleting of a layer can be found <<_configuring_map_for_intrigue_using_catalog_ui_search,
```

Comment:
✏️ nit: missing word "in" after "found" like in the other spot.

Reply:
Added the word and fixed wording of sentence.